app/index.html

```diff
@@ -16,7 +16,51 @@ <h1>HELLO WORLD!</h1><i class="fa fa-hand-spock-o" aria-hidden="true"></i>
   	<div class="header header__fifth"></div>
   	<div class="header header__sixth"></div>
   	<div class="header header__seventh"></div>
-  	<div class="header header__eighth"></div>
+  	<div class="header header__eighth">
+			<div class="container">
+				<nav class="nav">
+					<div class="wrapper">
+						<a href="#" class="nav__item"><img src="img/eighth_logo.png" alt="logo"></a>
+						<a href="#" class="nav__item">Home</a>
+						<a href="#" class="nav__item">Portfolio</a>
+						<a href="#" class="nav__item">Products</a>
+						<a href="#" class="nav__item">Services</a>
+						<a href="#" class="nav__item">About Us</a>
+						<a href="#" class="nav__item">Contact</a>
+					</div>
+					<form class="nav__search-form">
+						<div class="form-group">
+							<i class="fa fa-search form-group__icon"></i>
+							<input type="text" class="text-input" placeholder="Search" required>
+						</div>
+					</form>
+				</nav>
+				<div class="centered heading">WE MAKE PRODUCTS</div>
```

Comment:
please, rewrite title text with downcase because you've already user `text-transform: uppercase` in `heading` class